plugins/modules/hetzner_dns_record.py

```diff
@@ -31,6 +31,16 @@
     - community.dns.hetzner.zone_id_type
     - community.dns.module_record
     - community.dns.options.record_transformation
+    - community.dns.attributes
```

Comment:
(random line chosen for comment)
the rendered docs show a notes entry that should probably be removed: https://ansible.fontein.de/collections/community/dns/hetzner_dns_record_module.html#notes

I don't see it here, so I assume it's in a doc fragment?

Reply:
Yes, it came from there... Remove in 4ac21794668df9ee0c381243707cef9a606699cc. Thanks!